Assets/MRTK/SDK/Features/Utilities/Solvers/HandConstraintPalmUp.cs

```diff
@@ -90,6 +92,60 @@ public float FollowHandCameraFacingThresholdAngle
             set => followHandCameraFacingThresholdAngle = value;
         }
 
+        [SerializeField]
+        [Tooltip("With this active, solver will activate the UI after the palm threshold has been met and the user gazes at the activation point")]
+        private bool useGazeActivation = false;
+
+        /// <summary> 
+        /// With this active, solver will activate after the palm threshold has been met and the user gazes at the activation point
+        /// </summary>
+        public bool UseGazeActivation
+        {
+            get => useGazeActivation;
+            set => useGazeActivation = value;
+        }
+
+        [SerializeField]
+        [Tooltip("The distance between the planar intersection of the eye gaze ray and the activation transform. Uses square magnitude between two points for distance")]
+        [Range(0.0f, .1f)]
+        private float eyeGazeProximityThreshold = .005f;
+        /// <summary>
+        /// The distance threshold calculated between the planar intersection of the eye gaze ray and the activation transform. Uses square magnitude between two points for distance
+        /// </summary>
+        public float EyeGazeProximityThreshold
+        {
+            get => eyeGazeProximityThreshold;
+            set => eyeGazeProximityThreshold = value;
+        }
+
+        [SerializeField]
+        [Tooltip("The distance between the planar intersection of the head gaze ray and the activation transform. Uses square magnitude between two points for distance. This is used if eye gaze isn't available for the user")]
+        [Range(0.0f, .1f)]
+        private float headGazeProximityThreshold = .02f;
+
+        /// <summary>
+        /// The distance threshold calculated between the planar intersection of the head gaze ray and the activation transform. Uses square magnitude between two points for distance
+        /// This is used if eye gaze isn't available for the user
+        /// </summary>
+        public float HeadGazeProximityThreshold
+        {
+            get => headGazeProximityThreshold;
+            set => headGazeProximityThreshold = value;
+        }
+
+        private bool targetWorldLocked = false;
+
+        /// <summary>
+        /// Refects whether the current solver object is world-locked or not
```

Comment:
Is this the same as UpdateSolvers == false?

Reply:
Let me double-check; it's been a hot minute